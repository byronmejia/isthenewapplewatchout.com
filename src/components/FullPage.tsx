import { styled } from "@style/styled";

export const FullPage = styled.div`
    width: 100vw;
    height: 100vh;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
`;
