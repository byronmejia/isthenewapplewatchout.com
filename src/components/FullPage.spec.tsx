import { cleanup, render } from "@testing-library/react";
import * as React from "react";
import { FullPage } from "@components/FullPage";

describe("FullPage", () => {
    afterEach(cleanup);

    it("should render", () => {
        const output = render(<FullPage />);
        expect(output.baseElement).toBeTruthy();
        expect(output.baseElement).toMatchSnapshot();
    });
});
