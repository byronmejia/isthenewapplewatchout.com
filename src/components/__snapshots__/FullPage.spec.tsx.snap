// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullPage should render 1`] = `
.c0 {
  width: 100vw;
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: relative;
}

<body>
  <div>
    <div
      class="c0"
    />
  </div>
</body>
`;
